frontend/src/app/modules/availability/new-slot/event-detail/event-detail.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-event-detail',
+    templateUrl: './event-detail.component.html',
+    styleUrls: ['./event-detail.component.sass'],
+})
+export class EventDetailComponent {
+    timeZoneChoices: string[] = [
+        'Automatically detect and show the times in Bookers time zone',
+        'Lock the timezone (best for in-person events)',
+    ];
+
+    public zoneChoice: string;
+
+    public linkChoice: string = 'heorhim';
+
+    public welcomeMessage: string = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.';
+
+    public languageSelect: string = 'English';
+
+    public allowBookingSelect: string = 'two';
+
+    public isAllowBooker: boolean;
+
+    public basicChoice: boolean = true;
+
+    public passwordProtect: boolean = true;
+
+    public passwordInput: string = 'Meeting123';
```

Comment:
It would be better to create an interface for this entity (or extend existed) and use it here